cmd/bedFormat/testdata/expected.fdrAnnotation.bed

```diff
@@ -0,0 +1,96 @@
+chr1	0	10	chr1_0	2	+	2.000000e+01	1.019461e+00	5.423397e-01
```

Comment:
Can I ask what are the 3 columns after + (strand)? Are they all annotation? I thought there would be 2 columns of annotation, raw p value and adjusted p value?